### PR TITLE
Added join_group(url) to the group.py as per #53

### PR DESCRIPTION
**Pull request by shivam5522**
I have tried to create the function. Please check and revert if something is wrong. Thanks

### CODE REVIEW

**Comment by NavpreetDevpuri** [comment]
@shivam5522 good.
Try to run it.

File: tithiwa/group.py [hunk 1]
```diff
@@ -240,6 +240,14 @@ def _wait_for_group_info_to_load(self):
                 chatinfo = self._wait_for_presence_of_an_element(SELECTORS.CHATROOM__INFO).get_attribute('innerText')
             except:
                 pass
+    def join_group(self,url):
+        code_pos = url.index('com/')
+        code_pos += 4
+        code = url[code_pos:]
+        group_url = 'https://web.whatsapp.com/accept?code='+ code
+        driver.get(group_url)
```
Comment (NavpreetDevpuri) [inline tithiwa/group.py:248]: I suggest using pycharm ide 
`driver` is not defined in the scope. It should be `self.browser`

**Comment by shivam5522** [comment]
Would it be possible for you to run it because I am getting errors due to some modules and I can't understand how to go past them. I've run the function seperately in a python script and it works perfectly fine.

File: tithiwa/group.py [hunk 2]
```diff
@@ -240,6 +240,14 @@ def _wait_for_group_info_to_load(self):
...
+        time.sleep(10)
```
Comment (NavpreetDevpuri) [inline tithiwa/group.py:249]: using `sleep()` may create problems, I explained why we should avoid sleep() at this [PR](https://github.com/Tithibots/tithiwa/pull/63#issuecomment-714571161)

File: tithiwa/group.py [hunk 3]
```diff
@@ -240,6 +240,14 @@ def _wait_for_group_info_to_load(self):
...
+        driver.find_element_by_xpath('//*[@id="app"]/div/span[2]/div/div/div/div/div/div/div[2]/div[2]/div/div').click()
```
Comment (NavpreetDevpuri) [inline tithiwa/group.py:250]: Check this [suggestion](https://github.com/Tithibots/tithiwa/pull/63#discussion_r510258807)

**Review by NavpreetDevpuri — CHANGES_REQUESTED** [review]

**Comment by NavpreetDevpuri** [comment]
> Would it be possible for you to run it because I am getting errors due to some modules and I can't understand how to go past them. I've run the function seperately in a python script and it works perfectly fine.

Understood. No problem, I suggest you work on https://github.com/Tithibots/tithiwa/issues/66 because https://github.com/Tithibots/tithiwa/issues/66 is easier to get started.

File: tithiwa/group.py (hunk 2 above)
Comment (shivam5522) [reply tithiwa/group.py:249]: What would be the alternative then?

Comment (NavpreetDevpuri) [reply tithiwa/group.py:249]: by using `self._wait_for_presence_of_an_element(selector)`

**Comment by shivam5522** [comment]
I updated the errors again. Can you please check it out?

**Comment by NavpreetDevpuri** [comment]
You are new to automation just like was once.
Here are some tips for you 
1. While automating the task keep in mind every automation step takes time even milliseconds, so to take care of it we created `_wait_for` functions. 
2. Whenever trying to contribute to an open-source first of all look at the code base and try to understand similar things in the codebase and get an idea of the basic "structure of the codes". You will surprise how much you can learn by just trying to understand some code written by others. 
3. Always try to run and debug. Specially debug, I started debugging lately and I released if I had started debugging early then I may learn much more in less time.

good luck!

**Review by NavpreetDevpuri — APPROVED** [review]

**Comment by shivam5522** [comment]
Thank you for the tips!

**Pull request merged**